docs/proposals/backup-restore.md

```diff
@@ -121,18 +121,35 @@ oras backup --output backup.tar --include-referrers registry-a.k8s.io/kube-apise
 Alternatively, `oras` should also enables users to choose which tags within a repository to back up. For example, back up `v1` and `v2` tags with their referrers:
 
 ```bash
-oras backup registry-a.k8s.io/kube-apiserver:v1,v2 --include-referrers --output airgap-snapshot.tar
+oras backup registry-a.k8s.io/kube-apiserver:v1,v2 --include-referrers --output backup.tar
 ```
 
 Upon success, the output will be:
 
 ```console
-Found 2 tag(s) in registry-a.k8s.io/kube-apiserver
-- v1
-- v2
-<progress bar>...
-Exporting to backup.tar
-Successfully backed up 2 tag(s) from registry-a.k8s.io/kube-apiserver to backup.tar
+## <progress_bar>
+✔ Pulled tag v1 and 1 referrer(s)
+✓ Pulled  application/vnd.oci.image.config.v1+json                                                               2.26/2.26 KB  100.00%  447ms
+  └─ sha256:45a5868eb9f1dfbce42513000964664014789a43310865b0c8461e773e9972b9
+✓ Pulled  application/vnd.oci.image.layer.v1.tar+gzip                                                            25.6/25.6 MB  100.00%     4s
+  └─ sha256:149362fdfa6e6a5d9f009b896da3be3172c395ba2287b57d4969f3f46e573055
+✓ Pulled  application/vnd.cncf.notary.signature                                                                 1.85/1.85 MB  100.00%  898ms
+  └─ sha256:9b666bc868511a0f2d33a738a9ff0bd54eb750a72a832e8b59085d22bbdbaac2
+
+✔ Pulled tag v2 and 1 referrer(s)
+✓ Pulled  application/vnd.oci.image.config.v1+json                                                               2.24/2.24 KB  100.00%  353ms
+  └─ sha256:f9248aac10f2f82e0970222e36cc7b71215b88e974e001282e5cd89797a82218
+✓ Pulled  application/vnd.oci.image.layer.v1.tar+gzip                                                            28.3/28.3 MB  100.00%     3s
+  └─ sha256:b08e2ff4391ef70ca747960a731d1f21a75febbd86edc403cd1514a099615808
+✓ Pulled  application/vnd.cncf.notary.signature                                                                 1.85/1.85 MB  100.00%  890ms
+  └─ sha256:854ad9e87ce93dae54ae1699837b2c812d2f373c3fb62625ea6992efa8f023c4
+
+## <status output>
+Found 2 tag(s): v1, v2 in registry-a.k8s.io/kube-apiserver
+Pulled tag v1 and 1 referrer(s)
+Pulled tag v2 and 1 referrer(s)
+Exported to backup.tar (58.8 MB)
```

Comment:
It takes time to exporting to a tarball as well as loading from a tarball. Therefore, it is better to have **Exporting** and **Exported**.